packages/fast-components-class-name-contracts-base/src/horizontal-overflow/index.ts

```diff
@@ -2,8 +2,23 @@
  * The class name contract for the horizontal overflow component
  */
 export interface IHorizontalOverflowClassNameContract {
+    /**
+     * The root horizontal overflow component - contains overflow items and previous/next controls
+     */
     horizontalOverflow: string;
+
+    /**
+     * The overflow items
```

Comment:
This is a bit confusing - are these the items that shouldn't be displayed because they are overflowed?

Maybe we should change to "contentRegion" to align? That's really what this is